tests/unit/routers/test_entity.py

```
@@ -920,3 +934,400 @@ def test_get_entity_with_linked_local_plans(
         else:
             logging.warning("result has no context")
         assert False, "template unable to render, missing variable(s) from context"
+
+
+def _mock_search_entities_html_dependencies(
```

Pylint (4.0.4)

[refactor] Too many arguments (6/5)

(R0913)